deps/build.jl

```diff
@@ -0,0 +1,4 @@
+default_dir = get(ENV, "CONDA_JL_HOME", abspath(dirname(@__FILE__), "usr"))
```

Comment:
Why use an environment variable to specify the conda root path? I don't see the use case here.